pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -108,12 +108,6 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		}
 	}
 
-	// Create CSI config map
-	err = csi.CreateCsiConfigMap(c.OpManagerCtx, c.namespacedName.Namespace, c.context.Clientset, cluster.ownerInfo)
```

Comment:
Will the external cluster reconcile be failed at some point if the configmap doesn't exist?

Reply:
I looked into this, and I don't believe so. 

`pkg/operator/ceph/csi/controller.go` line 192 "gates" creation of the csi config map.  The only metric that affects creation is whether or not any CephClusters exist. It doesn't matter if the clusters are internal or external. By my logic, if an external cluster exists, the CSI controller will create the configmap (unless there is a create error), and so it is safe for the external reconcile to wait until that happens.

It might be good for you to double check my logic just to be sure.

Reply:
I see now that `SaveClusterConfig()` will be called below on line 133, but given your changes in that method, it will fail the reconcile and retry. Makes sense to me.